UnitsNet/CustomCode/Quantities/Length.extra.cs

```diff
@@ -80,7 +101,8 @@ public static bool TryParseFeetInches([CanBeNull] string str, out Length result,
             str = str.Trim();
 
             // This succeeds if only feet or inches are given, not both
-            if (TryParse(str, formatProvider, out result))
+            // Do not allow thousands separator here, since it may be equal to the unit abbreviation (').
+            if (TryParse(str, formatProvider, NumberStyles.AllowDecimalPoint | NumberStyles.AllowExponent | NumberStyles.AllowLeadingSign, out result))
```

Comment:
It seems from [docs](https://docs.microsoft.com/en-us/dotnet/api/system.double.tryparse?view=netcore-3.1#System_Double_TryParse_System_String_System_Double__) that `double.Parse()` defaults to `NumberStyles.Float` and `NumberStyles.AllowThousands`. Could we then simply use [NumberStyles.Float](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.numberstyles?view=netcore-3.1#System_Globalization_NumberStyles_Float
) here?

> The s parameter is interpreted by using a combination of the NumberStyles.Float and NumberStyles.AllowThousands flags. This means that white space and thousands separators are allowed but currency symbols are not.

Reply:
Yea, that's probably the better choice.